src/nonLinPMMH.cpp

```diff
@@ -97,6 +99,14 @@ Rcpp::DataFrame nonLinPMMH_impl(arma::vec data, unsigned long lNumber, unsigned
                 loglike(i) = loglike(i-1);
                 logprior(i) = logprior(i-1);
             }
+            if (i % 100 == 0 && verbouse){
+		Rcpp::Rcout << "========================="<< endl;
+		Rcpp::Rcout << "Iteration number: " << i << endl;
+		Rcpp::Rcout << "The MH_ratio is: " << std :: min(1.0, MH_ratio) << endl;
+		Rcpp::Rcout << "The MH_ratio_average is: " << MH_ratio_total / i << endl;
+		Rcpp::Rcout << "Logarithmic probability: " << loglike_prop << endl;
```

Comment:
I'm not sure you're outputting the right thing here; you'd ideally have the logarithm of the target density, not the logarithm of the likelihood of the most recently proposed value. That would amount to loglike(i) + logprior(i) and coul be described as _log posterior density_ (which it is up to normalization).